tensorflow/training/docker/1.15.3/py3/cu100/Dockerfile.gpu

```diff
@@ -176,7 +176,7 @@ RUN ${PIP} install --no-cache-dir -U \
     Pillow \
     h5py==2.10.0 \
     requests==2.22.0 \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```

Comment:
Please make this an ARG like the other dockerfiles

Reply:
Done.